src/stores/gitClone.js

```diff
@@ -0,0 +1,29 @@
+import { defineStore, mapActions } from 'pinia';
```

Comment:
I've organized most of the stores based around what type of record would be stored in their state. Since there isn't any state here I'm wondering if this would be handled better in the component. This works fine, and if we have "clone" specific state that we want to share across components, it would go here, I'm just not sure what that would be at the moment.